Systems Pharmacology

My research in this area primarily harnessed the molecular high-throughput data from drug treatment to systematically investigate the mechanism of action (MOA) for drugs. 
<br><br>
Research projects:
<br>
<ul>
	<li>Developed a multilabel classification approach to predict adverse drug reactions (ADRs) by utilizing the large-scale drug-induced gene expression profiles from the LINCS L1000 data and chemical structures. Article describing this approach is published in <i>Bioinformatics</i>. PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27153606" target="_blank">27153606</a>
		<br>A companion website visualizing the predicted ADRs for drugs is available at : <a href="http://maayanlab.net/SEP-L1000/"  target="_blank"> http://maayanlab.net/SEP-L1000/</a>
	</li>
	<li>Developed a graph-based dimensionality reduction algorithm to visualize tens of thousands drug-induced transcriptomics signatures. This approach facilitates the discovery of drug MOAs. It has been applied to transcriptomics datasets:
	<ul>
		<li><a href="http://amp.pharm.mssm.edu/LJP/" target="_blank">LINCS Joint Project (LJP)</a>: 2,300 expression signatures from six breast cancer cell lines treated with ~100 single molecule perturbations. An article describing this study has been published in <i>Nature Communications</i>. PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29084964" target="_blank">29084964</a></li>
		<li><a href="http://amp.pharm.mssm.edu/L1000FWD" target="_blank">L1000 Firework Display (L1000FWD)</a>: over 16,000 drug-induced gene expression signatures from the LINCS L1000 dataset. An article describing this study has been published in <i>Bioinformatics</i>. PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29420694" target="_blank">29420694</a></li>
	</ul>  
	</li>
</ul>



Translational Biomedical Informatics

I also focus on developing methods to mine the data from the Electronic Medical Records (EMR) to extract human phenotypes such as physiological health status and complex diseases. In our recent study (PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29113935" target="_blank">29113935</a>), we developed a deep learning regression model to estimate physiological age for over 300,000 patients using routine laboratory tests and vital signs. This approach discovered that there are patients with improved and decreased health status with regard to their age. 



Bioinformatics Resources, Tools and Pipelines for Data Integration 

I have developed resources and tools for re-analysis of public datasets and integrating knowledge extracted from across various resources for biomedical research.
<br><br>
Research projects: 
<br>
<ul>
	<li>CRowd Extracted Expression of Differential Signatures (<a href="http://amp.pharm.mssm.edu/CREEDS/" target="_blank">CREEDS</a>): a crowdsourcing approach to curate gene expression datasets from the NCBI's Gene Expression Omnibus (GEO) repository. An article describing the method and resultant resource has been published in <i>Nature Communications</i>. PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27667448">27667448</a>.</li>
	<li><a href="http://amp.pharm.mssm.edu/Enrichr">Enrichr</a>: a gene set enrichment analysis tool that includes one of the largest collections of annotated gene sets. </li>
	<li><a href="http://amp.pharm.mssm.edu/Harmonizome">Harmonizome</a>: an encyclopedia for genes and proteins that integrates curated knowledge and high quality experimental data. An article describing this resource has been published in <i>Database</i>. PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27374120">27374120</a></li>
	<li><a href="https://github.com/MaayanLab/Zika-RNAseq-Pipeline">An open and reproducible RNA-seq analysis pipeline</a>: this pipeline is able to automatically process RNA-seq data from scratch and generate comprehensive and interactive report. An article describing this pipeline has been published in <i>F1000Research</i>. PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27583132">27583132</a></li>
</ul>
